pkg/apiserver/resthandler.go

```diff
@@ -111,23 +118,83 @@ func getResourceHandler(scope RequestScope, getter getterFunc) restful.RouteFunc
 	}
 }
 
+func parseBool(str string) (bool, error) {
+	if len(str) == 0 {
+		return false, nil
+	}
+	return strconv.ParseBool(str)
+}
+
+func getExportOptions(req *restful.Request) (ExportOptions, error) {
```

Comment:
should likely do something like 
https://github.com/kubernetes/kubernetes/blob/master/pkg/apiserver/resthandler.go#L154